pkg/trace/filters/replacer.go

```diff
@@ -50,7 +50,9 @@ func (f Replacer) Replace(trace pb.Trace) {
 }
 
 // ReplaceStatsGroup applies the replacer rules to the given stats bucket group.
-func (f Replacer) ReplaceStatsGroup(b *pb.ClientGroupedStats) {
+func (f Replacer) ReplaceStatsGroup(b *pb.ClientGroupedStats) bool {
```

Comment:
Can you please update the documentation for these methods to specify what the return `bool` is?
```
... and reports whether the input was altered.
```
Or something like that.